internal/command/machine/run.go

```diff
@@ -194,6 +198,11 @@ func runMachineRun(ctx context.Context) error {
 		},
 	}
 
+	schedule := flag.GetString(ctx, "schedule")
```

Comment:
set it directly and drop the if 

```suggestion
	machineConf.Schedule = flag.GetString(ctx, "schedule")
```